private/tools/java/rules/jvm/external/javadoc/JavadocJarMaker.java

```diff
@@ -59,10 +59,11 @@ public static void main(String[] args) throws IOException {
     Set<Path> sourceJars = new HashSet<>();
     Path out = null;
     Set<Path> classpath = new HashSet<>();
+    List<String> options = new ArrayList<>();
 
     for (int i = 0; i < args.length; i++) {
       String flag = args[i];
-      String next = args[++i];
+      String next = i + 1 < args.length ? args[++i] : null;
```

Comment:
I'm guessing this is to handle the case where we may have single flags? If that's true, we should avoid reading `next` until we know that `flag` implies that there's a second parameter. That means that we should only be reading `next` if `flag` is one of `--cp`, `--in`, or `--out`